src/components/UI/StyleGuide.js

```diff
@@ -38,12 +39,56 @@ Header.propTypes = {
 
 // eslint-disable-next-line react/no-multi-comp
 const StyleGuide = () => {
+  const [selectedVersion, setVersion] = useState('1.0.5');
+
+  const handleSetVersion = version => {
```

Comment:
Why not letting the VersionPicker component call `setVersion` directly?

Reply:
This is like the callback that the VersionPicker exposes, that way whatever is implementing it can decide what to do when someone picks a version.

Reply:
But is there a difference in
```
<VersionPicker
  onChange={setVersion}
  ... />
```
and
```<VersionPicker
  onChange={handleSetVersion}
  ... />
```

Right now it just adds another function which just calls the `setVersion` function withougt changing parameters or anything

Reply:
Oh :) heh yeah. I guess I was just trying to show a pattern for the recommended implementation.